e2e/fixtures/monthOfYear.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  0: 'Jan',
```

Comment:
is the numbering definitely right for this file? i.e currentDate.getMonth() returns 0 for January?

Reply:
yes